routing/bicycle_directions.cpp

```diff
@@ -0,0 +1,187 @@
+#include "routing/bicycle_directions.hpp"
+#include "routing/car_model.hpp"
+#include "routing/router_delegate.hpp"
+#include "routing/routing_result_graph.hpp"
+#include "routing/turns_generator.hpp"
+
+#include "indexer/ftypes_matcher.hpp"
+#include "indexer/index.hpp"
+#include "indexer/scales.hpp"
+
+#include "geometry/point2d.hpp"
+
+namespace
+{
+using namespace routing;
+using namespace routing::turns;
+
+class AStarRoutingResult : public IRoutingResult
```

Comment:
Do not we use ISomeName for interfaces? At least it's very widespread in routing code. For example:
- IDirectionsEngine
- IOnlineFetcher
- IRoadGraph
- IRouter
- IRoutingAlgorithm
- IVehicleModel
- IVehicleModelFactory
as well.